bundle/Core/Search/RelatedContent/SortService.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Netgen\TagsBundle\Core\Search\RelatedContent;
+
+use eZ\Publish\API\Repository\Values\Content\Query;
+use eZ\Publish\API\Repository\Values\Content\Query\SortClause;
+use Netgen\TagsBundle\API\Repository\Values\Content\Query\SortClause\ContentTypeId;
+
+class SortService
```

Comment:
Probably a better name for this would be `SortClauseMapper` :)

Reply:
I named it `service` because it doesn't only map sort clauses but it also returns valid sort options (to be used in form dropdown element) so it's kind of a all-in-one sort service. Should I rename it anyways?

Reply:
Yes, please. To me, it doesn't matter where it is used, it's what it does. And in essence, it maps sort clauses.

Reply:
Fixed in https://github.com/netgen/TagsBundle/pull/98/commits/45691e2549bde8a5056a3fa8b1a80aca28825848.